src/model.rs

```diff
@@ -88,164 +88,190 @@ impl Default for ValTnsr {
+                        dtype: DataKind::Tnsr,
+                        val: 0,
+                        meta: mm,
+                    }
+                }
+            }
+
+            Mdl::Conv2d([stride_h, stride_w, pad, act, inpt, wght]) => {
+                // Check types
+                assert!(x(stride_h).dtype == DataKind::Scalar);
+                assert!(x(stride_w).dtype == DataKind::Scalar);
+                assert!(x(pad).dtype == DataKind::Scalar);
+                assert!(x(act).dtype == DataKind::Scalar);
+                assert!(x(inpt).dtype == DataKind::Tnsr);
+                assert!(x(wght).dtype == DataKind::Tnsr);
+
+                // Get arguments
+                let t_inpt = x(inpt).meta;
+                let t_wght = x(wght).meta;
+                let strideH = x(stride_h).val;
+                let strideW = x(stride_w).val;
+                let padding: PaddingMode = x(pad).val.try_into().unwrap();
+                let activation: ActiMode = x(act).val.try_into().unwrap();
+
+                // Create tensorhandle and get metadata
+                unsafe {
+                    //let start_time = Instant::now();
+                    let res = g.conv2d1(t_inpt, t_wght, strideH, strideW, padding, activation);
+                    //let duration = start_time.elapsed();
+                    //println!("  Time taken get conv: {:?}", duration);
+                    //let r_cost = (*(*res).op.ptr).runtime;
+                    //println!("Cost of conv2d is {}", r_cost);
+                    Self::Data {
+                        dtype: DataKind::Tnsr,
+                        val: 0,
+                        meta: res,
+                    }
+                }
+            }
+
+            Mdl::Ewadd([a, b]) => {
+                // Check types
+                assert!(x(a).dtype == DataKind::Tnsr);
+                assert!(x(b).dtype == DataKind::Tnsr);
+
+                // Get arguments
+                let t_a = x(a).meta;
+                let t_b = x(b).meta;
+
+                // Create tensorhandle and get metadata
+                unsafe {
+                    //let start_time = Instant::now();
+                    let res = g.element(OpType_OP_EW_ADD, t_a, t_b);
+                    //let duration = start_time.elapsed();
+                    //println!("  Time taken get ele: {:?}", duration);
+                    //let r_cost = (*(*res).op.ptr).runtime;
+                    //println!("Cost of ewadd is {}", r_cost);
+                    Self::Data {
+                        dtype: DataKind::Tnsr,
+                        val: 0,
+                        meta: res,
+                    }
+                }
+            }
+
+            Mdl::Relu(a) => {
+                assert!(x(a).dtype == DataKind::Tnsr);
+                let t_a = x(a).meta;
+
+                unsafe {
+                    let relu = g.relu(t_a, true);
+                    //let r_cost = (*(*relu).op.ptr).runtime;
```

Comment:
Remove? In general, we don't want to commit code that is commented out. Either remove it or use it.

Reply:
Thanks for pointing out. I removed all these commented code in the current branch I'm developing. It's a bit cumbersome to remove here and rebase another time later, so the next PR will have these cleaned up :)